### PR TITLE
Add parent method to Gollum::Git::Commit

### PR DESCRIPTION
**Pull request by heavywatal**
to fulfill a new spec: gollum/adapter_specs#11 required by gollum/gollum#1543

### CODE REVIEW

File: lib/rugged_adapter/git_layer_rugged.rb [hunk 1]
```diff
@@ -126,6 +126,10 @@ def stats
         @stats ||= build_stats
       end
 
+      def parent
+        Gollum::Git::Commit.new(@commit.parents.first)
```
Comment (dometto) [inline lib/rugged_adapter/git_layer_rugged.rb:130]: To take care of the 'no parent' edge case, I assume it should be something like this:

`@commit.parents.empty? ? nil : Gollum::Git::Commit.new(@commit.parents.first)`

Comment (heavywatal) [reply lib/rugged_adapter/git_layer_rugged.rb]: Thanks! Fixed accordingly.

**Comment by dometto** [comment]
@heavywatal I reran the Travis tests for this PR after merging your changes to the adapter specs, but now the tests are failing on the new spec you added:

**Comment by heavywatal** [comment]
You're right! The `first` seems to be the latest commit. Maybe this should be defined in a spec as well. And I found another problem: in rugged_adapter, `repo.commits` passes `max_count` option to `git.log`, and its default value is 10. It means that `repo.commits.last` is not always the root commit.

**Comment by dometto** [comment]
Finally got the right test going :) Thanks again @heavywatal! Before we can release this, we need to implement the same method for the RJGit adapter. Is that something you'd be interested in doing?

**Comment by heavywatal** [comment]
My pleasure! Thank you, too. I already did: https://github.com/gollum/rjgit_adapter/pull/18

**Pull request merged**